pkg/rancher-desktop/backend/lima.ts

```diff
@@ -1875,7 +1878,7 @@ CREDFWD_URL='http://${ hostIPAddr }:${ stateInfo.port }'
       throw ex;
     }
 
-    this.cfg = undefined;
+    this.cfg = defaultSettings;
```

Comment:
This seems like a bad idea: we can end up running with the wrong configuration. This was previously optional to ensure that we fall over if we ended up somehow trying to start the backend without setting the configuration.